hack/cluster-sync.sh

```diff
@@ -28,6 +28,7 @@ function main() {
     CLEAN_PID=$!
 
     make cluster-build
+    make manifests
```

Comment:
Including this will slow down all jobs running on the CI.

As I understand it, regenerating the manifests is only necessary if `KUBEVIRT_INFRA_REPLICAS!=2`...

So this is just a suggestion, but wouldn't it be better to run `make manifests` only if the number of infra replicas is different than 2?

Reply:
`make cluster-build` is only called by `make cluster-sync`, which had an `&& build-manifests` in its Makefile command, I just moved it here.